src/WebJobs.Script/Content/Script/http/response.js

```diff
@@ -6,50 +6,73 @@ module.exports = (context) => {
         headers: {},
         body: undefined,
 
+        // functions methods
+        raw: (body) => {
+            res.isRaw = true;
+            return res.send(body);
+        },
+
+        // node httpResponse methods
+        setHeader: (field, val) => {
```

Comment:
So you're adding set/get/remove for headers, in addition to the existing set method and array access. No objections just wondering what the motivation is for adding more ways to do the same thing?